htdocs/search.php

```diff
@@ -1108,17 +1120,11 @@
         }
 
         // additional options
-        if (!isset($options['f_userowner'])) {
-            $options['f_userowner'] = '0';
-        }
-        if ($options['f_userowner'] != 0) {  // Ocprop
+        if ($options['f_userowner'] === 1) {  // Ocprop
```

Comment:
@Haehnchen Feel free to join our development team and start migrating legacy code to the upcoming symfony app ;)